webapp/sources/ldap-inventory/inventory-provisioning-web/pom.xml

```diff
@@ -88,7 +88,6 @@ along with Rudder.  If not, see <http://www.gnu.org/licenses/>.
     <dependency>
```

Comment:
You can remove this dependency if added in parent

Reply:
In parent, I just added the dependency management (ie the version, so that all rudder use the same version). Each non-transitively depending project should still add the dependency.

Reply:
the dependency is added in parent, not just the version, and you didn't have to add it to rudder

Reply:
it's in <dependencyManagement>, not <dependency>

Reply:
ha ok